examples/material-ui/README.md

```diff
@@ -0,0 +1,53 @@
+# Welcome to Remix!
```

Comment:
Could you update this README to follow the format that's in the `examples/template/README.md`?